lib/ruby_lsp/tapioca/addon.rb

```diff
@@ -32,8 +32,7 @@ def initialize
       sig { override.params(global_state: RubyLsp::GlobalState, outgoing_queue: Thread::Queue).void }
       def activate(global_state, outgoing_queue)
         @global_state = global_state
-        # TODO: Uncomment
-        # return unless @global_state.experimental_features
+        return unless @global_state.enabled_feature?(:tapioca_addon)
```

Comment:
This is camel case because it comes from the JSON settings and we don't convert it to snake case.

```suggestion
        return unless @global_state.enabled_feature?(:tapiocaAddon)
```